docs/run-performance-metrics-collection-on-local.md

```diff
@@ -27,6 +27,9 @@ export TAF_COMMON_IMAGE=nexus3.edgexfoundry.org:10003/docker-edgex-taf-common-ar
 ### Run scripts
 ```
 cd $WORK_DIR/TAF/utils/scripts/docker
+sh get-compose-file-perfermance.sh ${USE_ARCH}
```

Comment:
Isn't this already included in the exec_performance_metrics.sh?
https://github.com/edgexfoundry/edgex-taf/blob/fca51c6c30dd99333d5f391532d3a040ad01b0e5/TAF/utils/scripts/docker/exec_performance_metrics.sh#L6-L7

Reply:
Remove the change.